tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -289,9 +340,7 @@ class MbedTLSCli(TLSProgram):
 
     def cmd(self):
         super().cmd()
-        ret = ['$P_CLI']
-        ret += ['server_addr=127.0.0.1', 'server_port=$SRV_PORT',
-                'debug_level=4']
+        ret = ['server_addr=127.0.0.1', 'server_port=$SRV_PORT', 'debug_level=4']
```

Comment:
remove line 292. and
```suggestion
        ret = super().cmd()
        ret += ['server_addr=127.0.0.1', 'server_port=$SRV_PORT', 'debug_level=4']
```